src/analog/commands/downlink/SetParameter.ts

```diff
@@ -159,6 +159,197 @@ const examples: TCommandExampleList = [
             }
         },
         hex: {header: '03 05', body: '21 8c a0 65 90'}
+    },
+    {
+        name: 'set configuration for session',
```

Comment:
somewhere in name should be mqtt

and in next examples

Reply:
++